zerver/webhooks/beeminder/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Beeminder notifications in Zulip whenever you're going to derail from your goal!
+
+{!create-stream.md!}
+
+The webhook URL should look like:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the URL above where `api_key` is the API key of your Zulip bot
+and `email` is your Zulip email.
+
+* When creating a new goal in Beeminder, you can check the **Make private** option to let
+the bot send you a PM, otherwise, the bot will send a message to a public stream.
```

Comment:
I would change the punctuation and wording here slightly:

> When creating or editing a goal in Beeminder, you can check the **Make private** option to make sure the bot sends you private messages; otherwise, the bot will send messages to a public stream.